packages/paste-website/src/component-examples/RadioButtonGroupExamples.ts

```diff
@@ -0,0 +1,61 @@
+export const BasicExample = `const RadioButtonGroupExample = () => {
+  return (
+    <RadioButtonGroup attached name="foo">
+      <RadioButton value="1">One</RadioButton>
+      <RadioButton value="2" defaultChecked>Two</RadioButton>
+      <RadioButton value="3">Three</RadioButton>
+    </RadioButtonGroup>
+  )
+}
+
+render(<RadioButtonGroupExample />)`.trim();
+
+export const TextExample = `const RadioButtonGroupExample = () => {
+  return (
+    <RadioButtonGroup attached name="foo">
+      <RadioButton value="m">Mon</RadioButton>
+      <RadioButton value="t" defaultChecked>Tues</RadioButton>
+      <RadioButton value="w">Wed</RadioButton>
+      <RadioButton value="th">Thu</RadioButton>
+      <RadioButton value="f">Fri</RadioButton>
+    </RadioButtonGroup>
+  )
+}
+
+render(<RadioButtonGroupExample />)`.trim();
+
+export const TextAndIconExample = `const RadioButtonGroupExample = () => {
+  return (
+    <RadioButtonGroup attached name="foo">
+      <RadioButton value="phone"><CallIcon decorative={false} title="phone icon"/>Phone</RadioButton>
+      <RadioButton value="text"><SMSCapableIcon decorative={false} title="text icon"/>Text</RadioButton>
+      <RadioButton value="email"><EmailIcon decorative={false} title="email icon"/>Email</RadioButton>
+    </RadioButtonGroup>
+  )
+}
+
+render(<RadioButtonGroupExample />)`.trim();
+
+export const ErrorExample = `const RadioButtonGroupExample = () => {
+  return (
+    <RadioButtonGroup attached name="foo" legend="What is your preferred programming language?" errorText="Choose a programming language">
```

Comment:
suggestion: could we use the Text component as the legend here (like in the stories) so that we don't get the spacing issue?

Reply:
I discovered that the spacing issue is present whether `legend` is a string or a <Text />. The only way to fix it is by also including `helpText`. This might be a bug because there are probably use cases for including a legend but no help text.